### PR TITLE
Improvements to Simulation

### PR DESCRIPTION
**Pull request by pixelfelon**
Requires new FRCSim (see https://github.com/cpostbitbuckets/FRCSim/pull/7)

### CODE REVIEW

File: InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/auto/AutoDrive.java [hunk 1]
```diff
@@ -68,6 +68,7 @@ public void execute() {
     public boolean isFinished() {
         // whether the RAMSETE command trajectory has been executed
         boolean trajDone = super.isFinished();
+        trajDone = false;
```
Comment (cpostbitbuckets) [inline InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/auto/AutoDrive.java]: Can we drop this? It seems like a change we don't want in the mainline.

File: InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/RamseteSpecial.java [hunk 2]
```diff
@@ -0,0 +1,175 @@
+/*----------------------------------------------------------------------------*/
```
Comment (cpostbitbuckets) [inline InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/RamseteSpecial.java]: Is this file being used, or was it just for testing?

Comment (pixelfelon) [reply InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/RamseteSpecial.java]: No, this was just a test for running the controller past the time-limit of the trajectory.

File: InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/auto/AutoDrive.java (hunk 1 above)
Comment (pixelfelon) [reply InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/auto/AutoDrive.java]: Yep, I thought this got removed when I merged in from the master repo.

**Pull request merged**